content/en/docs/cloud/_index.md

```diff
@@ -28,10 +28,10 @@ overlay [virtual application network](https://netprototalk.com/2019/11/12/virtua
 4. **Developer familiarity**: feels like the frameworks and tools you already know and love (Docker,
    Kubernetes, Terraform, GitOps, etc).
 
-Built by developers for developers, we focus on beautifully crafted interfaces (web and cli),
+Built by developers for developers, we focus on functional interfaces (web and CLI),
```

Comment:
“functional” is a low bar (and sounds a bit boring) I like “…beautifully crafted interfaces…” but we could come up with some other options if you hate it.